vimeo/VimeoNetworking
# Add shared VimeoNetworking scheme to support Carthage

#### Pull Request Checklist


#### Issue Summary
Adding support for Carthage requires creating a shared scheme that builds a suitable target. 


#### Implementation Summary
- Shared the existing "VimeoNetworking-iOS" scheme and renamed it to "VimeoNetworking." This was enough to get it.
- Because the Pods project is managed by CocoaPods it's possible that a future update could alter or destroy the shared scheme. I'm not really sure what the best way around this is.


#### Reviewer Tips
- I'm not sure if creating a scheme from the Pods project is the most stable way to do this. The alternative seemed to be adding all of the VimeoNetworking files to the example project and creating a new target and scheme. But that would break the CocoaPods integration.



#### How to Test
- In a sample project add the following lines to a `Cartfile`:
```
github "Vimeo/VimeoNetworking" "tech/add-carthage-support"
github "AFNetworking/AFNetworking" == 3.1.0
```
- Follow the instructions in the Carthage [Getting Started](https://github.com/Carthage/Carthage#getting-started) guide.
- The result should be a successful integration of VimeoNetworking without the overhead of CocoaPods.
- A sample project with VimeoNetworking integration via Carthage can be found [here](https://www.dropbox.com/s/6tl5lggq3v7boym/Carthage%20SampleApp.zip?dl=0).
